Symfony/CS/LintManager/LintManagerInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\LintManager;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * Interface for PHP code linting process manager.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
```

Comment:
Do we have to make this interface internal. It would be cool to have some public interfaces in 2.0.